PriceKataCalc/Program.cs

```diff
@@ -5,35 +5,52 @@ class priceCalc
     {
         public static void Main()
         {
-            product myProduct = new("ball", 20405, 15.1567,0.20,0.15);
-            Console.WriteLine($"Product price reported as {myProduct.price} before tax and {myProduct.calcPriceAfterTax()} after {myProduct.tax*100}% tax, And with {myProduct.discount*100}% discont its {myProduct.calcPriceAfterDiscount()}  ");
+            product myProduct = new("ball", 20405, 15.1567, 0.20, 0.15, true);
+            Console.WriteLine($"Product price reported as {myProduct.price} before tax and {myProduct.calcPriceAfterTax()} after {myProduct.tax * 100}% tax  ");
+            if (myProduct.haveDiscount)
```

Comment:
Again, I would expect at this point this to be moved to a sperate class or method.